packages/genesys-spark-components/src/i18n/check-regional-dates-cookie.ts

```diff
@@ -0,0 +1,5 @@
+export const readRegionalDatesCookie = () => {
+  return !!document.cookie
+    .split('; ')
+    .find(cookie => cookie.startsWith('spark-enable-regional-dates'));
+};
```

Comment:
Why did you decide to feature toggle with a cookie over say a property on the component? Who do you see using this toggle?

Reply:
Matt suggested using a cookie as the feature toggle in the ticket: https://inindca.atlassian.net/browse/COMUI-2435